app/components/standup_meeting/meeting_update_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="border p-4 flex flex-col lg:flex-row justify-between mt-4">
+  <span class="flex flex-col justify-center mr-4 font-bold"><%= user.email %></span>
+  <span class="grow">
+    <%= standup_meeting.send(content_type) %>
```

Comment:
hope this level of metaprogramming is alright.

Reply:
My general rule is you should almost never use `send` (exception being inside gems for the most part). You should use `public_send`. For what you are doing i'd honestly move this into a method inside of the component ruby file.

One general alternative to this I like is just having a hash and doing fetch while passing in some param (you can memoize the hash, and it is pretty descriptive in what you are doing).

Reply:
lol. Yeah -- I was on the ropes, which is why I wanted to put this up as a draft. I did have some methods for doing it like a hash, but it introduced a lot more complexity into the component, so I wasn't sure. Anyway, will revisit tomorrow.